java/api/src/main/java/org/ray/api/runtime/RayRuntime.java

```diff
@@ -137,7 +137,7 @@
    * @param args Arguments of the function.
    * @return The result object.
    */
-  RayObject callPy(RayPyActor pyActor, String functionName, Object[] args);
+  RayObject callPyActor(RayPyActor pyActor, String functionName, Object[] args);
```

Comment:
After https://github.com/ray-project/ray/pull/7462 is merged, we can simply rename it to `call` .

Reply:
callPyActor is similar to callActor